basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -487,7 +510,7 @@ namespace basic_autonomy
             back_and_future.reserve(points_set.size());
             double total_dist = 0;
             int min_i = 0;
-            for (int i = nearest_pt_index; i > 0; --i)
+            for (int i = nearest_pt_index; i >= 0; --i)
```

Comment:
Can you add the explanation comment from my pr
```
// int must be used here to avoid overflow when i = 0
```

Reply:
Added comment